atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/DbKvs.java

```diff
@@ -1381,7 +1381,7 @@ private <T> void runWriteForceAutocommit(TableReference tableRef, Function<DbWri
             if (!autocommit) {
                 runWriteFreshConnection(conns, tableRef, runner);
             } else {
-                runner.apply(dbTables.createWrite(tableRef, conns));
+                T unused = runner.apply(dbTables.createWrite(tableRef, conns));
```

Comment:
The update caused this compile error ([circle](https://app.circleci.com/pipelines/github/palantir/atlasdb/13600/workflows/18c94aa9-0b22-43d1-aa81-9fec495580f1/jobs/56462)):
```
error: [CheckReturnValue] The result of `apply(...)` must be used
```

Think this is a result of updating the transitive `org.checkerframework` dependency, but haven't fully digged into what changed here tbh.

Reply:
This is ok, though I think it's preferable to add a `@SuppressWarnings("CheckReturnValue")` at the top of the method and leave a comment. In this case it's fine because this operation does a non-conditional write.

Reply:
Updated to suppress the warning instead.